public/stylesheets/results.css

```diff
@@ -0,0 +1,11 @@
+.correct {
```

Comment:
Through these class names alone I struggle to understand exactly which HTML elements these classes are applied to, except for `results-box`. I think these could use more descriptive names

Reply:
Will specify the names for these. I have also been considering moving the `correct` and `wrong` classes to card.css and reusing the classes from that stylesheet to display results instead.